src/shared_realm.hpp

```diff
@@ -155,6 +155,12 @@ class Realm : public std::enable_shared_from_this<Realm> {
         bool automatic_change_notifications = true;
     };
 
+    // Sets a path to a directory where Realm can write named pipes.
+    // This string should include a trailing slash '/'.
+    static void set_named_pipe_directory(std::string named_pipe_directory); // throw
```

Comment:
`// throws` comments are used at the callsite. The lack of `noexcept` here indicates that it throws so there's no need to explicitly note that.